src/client/localplayer.cpp

```diff
@@ -106,8 +106,8 @@ bool LocalPlayer::updateSneakNode(Map *map, const v3f &position,
 		f32 distance_f = diff.getLength();
 
 		if (distance_f > min_distance_f ||
-				fabs(diff.X) > (.5 + .1) * BS + sneak_max.X ||
-				fabs(diff.Y) > (.5 + .1) * BS + sneak_max.Z)
+				fabs(diff.X) > 0.6f * BS + sneak_max.X ||
+				fabs(diff.Y) > 0.6f * BS + sneak_max.Z)
```

Comment:
Intentional. `0.5` indicates 1/2 of a regular sized node. `+ 0.1` is for tolerances and sneaking on the edge.

Reply:
Ah ok, I was wondering what this was all about. Done, thanks.